producers/mysql-producer/src/main/java/com/flipkart/aesop/runtime/producer/eventprocessor/impl/DeleteEventProcessor.java

```diff
@@ -46,9 +51,19 @@ public void process(BinlogEventV4 event, OpenReplicationListener listener) throw
 		}
 		LOGGER.debug("Delete Event Received : " + event);
 		DeleteRowsEvent deleteRowsEvent = (DeleteRowsEvent) event;
+
+		List<Row> rowList = deleteRowsEvent.getRows();
+		List<Pair<Row>> listOfPairs = new ArrayList<Pair<Row>>(rowList.size());
+
+		for (Row row : rowList)
+		{
+			Pair rowPair = new Pair(null, row);
```

Comment:
Use proper generics. Also, remove unnecessary few usage declaration of variables.

for(Row row : deleteRowsEvent.getRows()) {

    listOfPairs.add(new Pair<>(null, row));
}